local-otel-configuration/prometheus.yml

```
@@ -0,0 +1,8 @@
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: 'otel-collector'
+    static_configs:
+      - targets: ['otel-collector:8889'] 
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)